app/src/main/java/org/mozilla/fenix/settings/quicksettings/WebsiteInfoView.kt

```diff
@@ -23,10 +21,12 @@ import org.mozilla.fenix.R
  */
 class WebsiteInfoView(
     container: ViewGroup
-) : LayoutContainer {
-
-    override val containerView: View = LayoutInflater.from(container.context)
-        .inflate(R.layout.quicksettings_website_info, container, true)
+) {
+    val binding = QuicksettingsWebsiteInfoBinding.inflate(
+        LayoutInflater.from(container.context),
+        container,
+        true
```

Comment:
@Amejia481 `attachToParent` was set to false the last time, that is why the panel did not show up.

Reply:
👍🏽